### PR TITLE
Input Plugin: Adding TTL field for Ping plugin 

### PR DESCRIPTION
**Pull request by dheerajdwivedi**
Feature request [5537](https://github.com/influxdata/telegraf/issues/5537)

### Required for all PRs:

- [x] Signed [CLA](https://influxdata.com/community/cla/).
- [x] Associated README.md updated.
- [x] Has appropriate unit tests.

### CODE REVIEW

File: plugins/inputs/ping/ping.go [hunk 1]
```diff
@@ -253,50 +257,75 @@ func (p *Ping) args(url string, system string) []string {
...
+				return trans, recv, ttl, min, avg, max, stddev, err
 			}
 		}
 	}
-	return trans, recv, min, avg, max, stddev, err
+	return trans, recv, ttl, min, avg, max, stddev, err
+}
+
+func getPacketStats(line string, trans, recv int) (int, int, error) {
+
```
Comment (glinton) [inline plugins/inputs/ping/ping.go]: Nitpick, remove this empty line.

Comment (dheerajdwivedi) [reply plugins/inputs/ping/ping.go]: Done

**Comment by danielnelson** [comment]
Let's call the new field just `ttl`.

**Comment by dheerajdwivedi** [comment]
> Let's call the new field just `ttl`.

Agreed and done.

That was a quick hack I had to do in last commit as build failed due to format check.
Not sure why format check passed in my local and failed while build.
Refer this [commit].(https://github.com/influxdata/telegraf/pull/5556/commits/a846039b43738be7f6e051e40b8792dfb10ff87e)

**Comment by dheerajdwivedi** [comment]
@danielnelson Looks like a false alarm. Can you please re-trigger the build.

**Pull request merged**